src/connector/src/parser/unified/json.rs

```diff
@@ -181,6 +181,22 @@ impl JsonParseOptions {
         struct_handling: StructHandling::AllowJsonString,
         ignoring_keycase: true,
     };
+    pub const DYNAMODB: JsonParseOptions = JsonParseOptions {
+        bytea_handling: ByteaHandling::Standard,
+        time_handling: TimeHandling::Micro,
+        timestamptz_handling: TimestamptzHandling::GuessNumberUnit, // backward-compatible
+        json_value_handling: JsonValueHandling::AsValue,
+        numeric_handling: NumericHandling::Relax {
+            string_parsing: true,
+        },
+        boolean_handling: BooleanHandling::Relax {
+            string_parsing: true,
+            string_integer_parsing: true,
+        },
+        varchar_handling: VarcharHandling::Strict,
+        struct_handling: StructHandling::Strict,
+        ignoring_keycase: true,
+    };
```

Comment:
Did you verify the configs are correct? (At least the "backward-compatible" comment is wrong 😂)

Reply:
Better to add some comments to demonstrate each choice.

Reply:
Mostly no specific reasons, as we don't really need to parse the data and we cannot predict the real formats of the data: in dynamodb and kinesis, they are just strings. The only thing I ensured is all these handling modes accept strings as input.

Reply:
If you don't understand what these configs do, and why do we need a specific config, perhaps you should just use `DEFAULT`. @xiangjinwu how do you think?

Reply:
I just took a quick attempt. `DEFAULT` work as well.